extensions/ql-vscode/src/vscode-tests/cli-integration/queries.test.ts

```diff
@@ -157,10 +157,11 @@ describe('Queries', function() {
   });
 
   it('should avoid creating a quick query', async () => {
+    fs.mkdirpSync(path.dirname(qlpackFile));
```

Comment:
No idea why this worked before.

Reply:
It works when you run all the tests together because previous tests create the directory. But if you run this test separately, it fails.